pkg/ccl/streamingccl/replicationtestutils/testutils.go

```diff
@@ -263,13 +263,18 @@ ORDER BY created DESC LIMIT 1`, c.Args.DestTenantName)
 // stop eventually. If the provided cutover time is the zero value, cutover to
 // the latest replicated time.
 func (c *TenantStreamingClusters) Cutover(
-	producerJobID, ingestionJobID int, cutoverTime time.Time, async bool,
+	ctx context.Context, producerJobID, ingestionJobID int, cutoverTime time.Time, async bool,
 ) {
 	// Cut over the ingestion job and the job will stop eventually.
 	var cutoverStr string
 	if cutoverTime.IsZero() {
 		c.DestSysSQL.QueryRow(c.T, `ALTER TENANT $1 COMPLETE REPLICATION TO LATEST`,
 			c.Args.DestTenantName).Scan(&cutoverStr)
+		cutoverOutput := DecimalTimeToHLC(c.T, cutoverStr)
+		// Fetch PTS on the source cluster
```

Comment:
nit: i think the docstring could instead explain why we check this, instead of explaining what the code does.